includes/tools.php

```diff
@@ -223,3 +226,145 @@ function verify_https_support() {
 	 */
 	return 200 === wp_remote_retrieve_response_code( $response );
 }
+
+/**
+ * Build a system environment report for submission along with any bug reports.
+ *
+ * This is largely based on (a simplified version) of Easy Digital Downloads' edd_tools_sysinfo_get()
+ * reporting function.
+ *
+ * @link https://github.com/easydigitaldownloads/easy-digital-downloads
```

Comment:
Would the inclusion of this as a derivative work change the license to GPL 2.0+?